src/components/TimerButton/WorkBreakButton.tsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from "react";
+import socket from "../socket";
+
+const WorkBreakButton = ({ roomName }: { roomName: string }): JSX.Element => {
+	// It may be needed to fetch from the server whether the user is on a break or not.
+	// The default isBreak state below is just a placeholder for now.
+	const [isBreak, setBreak] = useState(false);
```

Comment:
Can this isBreak useState be moved to the Room.tsx component and be passed down as a prop instead?

This is because to set the state for this, we will need the socket object, which is all in the room component.